--- yaml-cpp-config.cmake	2025-05-23 23:46:50.224298521 +0000
+++ yaml-cpp-config.cmake.patched	2025-05-23 23:47:09.990260059 +0000
@@ -44,14 +44,14 @@
 set(YAML_CPP_LIBRARIES "yaml-cpp::yaml-cpp")
 
 # Protect against multiple inclusion, which would fail when already imported targets are added once more.
-if(NOT TARGET yaml-cpp)
-  add_library(yaml-cpp INTERFACE IMPORTED)
-  target_link_libraries(yaml-cpp INTERFACE yaml-cpp::yaml-cpp)
-  if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.17)
-    set_target_properties(yaml-cpp PROPERTIES
-      DEPRECATION "The target yaml-cpp is deprecated and will be removed in version 0.10.0. Use the yaml-cpp::yaml-cpp target instead."
-    )
-  endif()
-endif()
+#if(NOT TARGET yaml-cpp)
+#  add_library(yaml-cpp INTERFACE IMPORTED)
+#  target_link_libraries(yaml-cpp INTERFACE yaml-cpp::yaml-cpp)
+#  if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.17)
+#    set_target_properties(yaml-cpp PROPERTIES
+#      DEPRECATION "The target yaml-cpp is deprecated and will be removed in version 0.10.0. Use the yaml-cpp::yaml-cpp target instead."
+#    )
+#  endif()
+#endif()
 
 check_required_components(yaml-cpp)
